setup/install/providers/ecs.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: single
+title:  "ECS (Amazon Web Services)"
+sidebar:
+  nav: setup
+redirect_from: /setup/providers/ecs/
+---
+
+{% include toc %}
+
+In the ECS cloud provider, an [__Account__](/concepts/providers/#accounts)
+maps to a Spinnaker AWS account, which itself is able to authenticate against a given [AWS
+account](https://aws.amazon.com/account/).
+
+## Prerequisites
+
+### ECS Cluster
+You need to [create an ECS cluster](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/create_cluster.html) and have enough capacity in it to deploy your containers.
+Please note that ECS clusters are not the same concept as [Spinnaker clusters](/concepts/clusters/).  
+
+### Networking 
+As Elastic Network Interfaces (ENIs) are not yet supported in Spinnaker, you do not need to setup any further networking.  The cluster's networking configuration will be passed from your cluster instances to your containers.   
+
+### Spinnaker Clouddriver role 
+
+The role that Clouddriver assumes for your ECS account needs to have the following trust relationships:
+```yaml
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+                "Service": [
+                  "ecs.amazonaws.com",
+                  "application-autoscaling.amazonaws.com"
+                ],
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+```  
+
+For information on how to set up the role Clouddriver assumes, see the [AWS documentation](/setup/install/providers/aws/#adding-an-account)
+
+### Optional setups
+
+You may create IAM roles that have the `ecs-tasks.amazonaws.com` trust relationship so that your containers have an IAM role associated to them.  
+
+```yaml
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "",
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "ecs-tasks.amazonaws.com"
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+```
+
+You may create IAM roles that have the `ecs-tasks.amazonaws.com` trust relationship so that your containers have an IAM role associated to them.  
+
+```yaml
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "",
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "ecs-tasks.amazonaws.com"
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
```

Comment:
Looks like these are all edited on the Amazon Console? You might add the steps for getting to them, as is done in the [existing AWS provider setup docs](https://www.spinnaker.io/setup/install/providers/aws/).